app/grants/views.py

```diff
@@ -2277,3 +2282,18 @@ def add_grant_from_collection(request, collection_id):
     return JsonResponse({
         'grants': grants,
     })
+
+
+@login_required
+@staff_member_required
+def collection_thumbnail(request, collection_id):
+    width = int(request.GET.get('w', 600))
+    height = int(request.GET.get('h', 400))
+    collection = GrantCollection.objects.get(pk=collection_id)
```

Comment:
^ try + catch ?

Reply:
IMHO since this method is for development purposes is better to show the exceptions instead of caching them.